ingest/source-data/nextclade-field-map.tsv

```diff
@@ -0,0 +1,17 @@
+key	value
+index	index
```

Comment:
Do we need the `index` column? From what I can see it gets filtered out in the join and is not used as a join key either.

Reply:
Dropped in https://github.com/nextstrain/monkeypox/pull/207/commits/0ff67dfdd4a967f055c22923386de7e05aca1a33